lib/api/apiUtils/object/coldStorage.js

```diff
@@ -0,0 +1,99 @@
+const ObjectMDArchive = require('arsenal').models.ObjectMDArchive;
+const errors = require('arsenal').errors;
+const { config } = require('../../../Config');
+const { locationConstraints } = config;
+
+
+/**
+ * Check if restore can be done.
+ *
+ * @param {BucketInfo} bucketMD - bucket instance
+ * @param {ObjectMD} objectMD - object metadata
+ * @return {Error|boolean} - true if the object is already restored
+ */
+function _validateStartRestore(bucketMD, objectMD) {
+    const isLocationCold = locationConstraints[bucketMD.getLocationConstraint()]?.isCold;
+    if (isLocationCold === undefined || isLocationCold === false) {
```

Comment:
A falsy value could be simplified with:
```suggestion
    if (!isLocationCold) {
```